src/client/features/view/components/menu/index.js

```diff
@@ -3,6 +3,11 @@ const h = require('react-hyperscript');
 const Link = require('react-router-dom').Link;
 const classNames = require('classnames');
 
+const layoutConf = require('../../../../common/cy/layout');
+
+const Dropdown = require('../../../../common/dropdown').Dropdown;
```

Comment:
Pretty sure you can do this:
```js
const { Dropdown, DropdownOption } = require('../../../../common/dropdown');
```

Reply:
Yup, that works. I've made that change and I'll keep in mind to change others when I come across them.